site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -70,9 +70,31 @@ The specified Lambda Id must be of type `SAMLv2Reconcile`.
 
 include::_identity-provider-linking-strategy-request-parameter.adoc[]
 
+[field]#identityProvider.loginHintConfiguration.enabled# [type]#[boolean]# [optional]#Optional#::
+When enabled, at the onset of the AuthN request (using redirect bindings), FusionAuth will attempt to pass a custom specified login hint to the SAML IdP.
+
+[field]#identityProvider.loginHintConfiguration.parameterName# [type]#[String]#::
+When using a login hint, at the onset of the AuthN request (using redirect bindings), the name of the login hint parameter to be provided to the SAML IdP.
+
 [field]#identityProvider.name# [type]#[String]# [required]#Required#::
 The name of this SAML v2 identity provider. This is only used for display purposes. Must be unique across all identity providers.
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.nameIdFormat# [type]#[String]# [required]#Required# [default]#Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`#::
+The `nameid-format` of this SAML v2 identity provider.
++
+Possible values are:
++
+- `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` -
+FusionAuth will expect the `NameId` returned will not change and is unique. When not specified, this is the default value.
++
+- `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress` -
+FusionAuth will expect an email address returned for the `NameId`. When setting this option, ensure email addresses within the external identity provider are globally unique and unchanging. If the external system does not have unique email addresses, identify a unique Id from the external IdP using the the `identityProvider.uniqueIdClaim` field.
++
+- `urn:oasis:names:tc:SAML:1.1:nameid-format:custom` -
+Allows customization of the `nameIdFormat` to be any value necessary. For instance, using this option, it is possible to set a nameIdFormat to `foo` per business requirements.
```

Comment:
I wouldn't mention `foo` as a custom option, this will still need to be a legit nameId. And This is not how you specify a custom nameId format.